Applications/spire/source/security_input/security_input_dialog.cpp

```diff
@@ -117,6 +115,14 @@ void SecurityInputDialog::mouseReleaseEvent(QMouseEvent* event) {
   m_is_dragging = false;
 }
 
+void SecurityInputDialog::showEvent(QShowEvent* event) {
+  m_security_input_box = new SecurityInputBox(Ref(*m_model), m_initial_text, this);
+  m_security_input_box->connect_commit_signal(
+    [=] (const Security& s) { set_security(s); });
```

Comment:
```(const auto& security)```

Reply:
I thought lambdas was one of the places we used single character names?